static/templates/settings/alert_word_settings_item.hbs

```diff
@@ -1,5 +1,5 @@
 {{! Alert word in the settings page that can be removed }}
```

Comment:
Let us rename this file as it currently contains the whole list not a single item, probably something like `alert_words_settings_list`.

Reply:
i don't think it contains the whole list, it does only contain a table row!

which is rendered through `ListWidget` in `alert_words_ui.js`

Reply:
so i think it's correctly named as `alert_word_settings_item`

Reply:
Yes, I misunderstood it.